go.mod

```diff
@@ -47,6 +47,7 @@ require (
 )
 
 require (
+	github.com/kedacore/keda/v2 v2.7.0
```

Comment:
Please re-base and use latest version 2.14.0. We managed to bump operators so now we have to support the latest version